cmd/gateway/validation_test.go

```diff
@@ -122,6 +128,133 @@ func TestValidateResourceName(t *testing.T) {
 	}
 }
 
+func TestValidateNamespaceName(t *testing.T) {
+	tests := []struct {
+		name   string
+		value  string
+		expErr bool
+	}{
+		{
+			name:   "valid",
+			value:  "mynamespace",
+			expErr: false,
+		},
+		{
+			name:   "valid - with dash",
+			value:  "my-namespace",
+			expErr: false,
+		},
+		{
+			name:   "valid - with dash and numbers",
+			value:  "my-namespace-123",
```

Comment:
Could this test be updated to use only numbers but no dashes? That way we aren't failing due to the dashes (since we test this previously) and know that we're testing the numbers.